.circleci/config.yml

```diff
@@ -21,6 +21,10 @@ executors:
 
 jobs:
   setup-multi-product-repositories:
+    parameters:
+      target-branch:
+        type: string
+        default: '$(curl -s -H "Authorization: token ${GITHUB_TOKEN}" $(echo https://api.github.com/repos/${CIRCLE_PULL_REQUEST:19} | sed "s/\/pull\//\/pulls\//") | jq ".base.ref" | tr -d "\042" )'
```

Comment:
Can you clarify what 19 is? I am assuming it's a substring, just want to check. Also what is `\042`?

Reply:
Yeah it's the substring, we just remove the `https://github.com/` part from the URL. And `\042` is the quotes to be removed with `tr -d` from the branch name.